apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.0__create_tables.sql

```diff
@@ -29,10 +29,11 @@ CREATE TABLE `challenge_input_data_type`
 CREATE TABLE `challenge`
 (
     `id`                    bigint(20) NOT NULL AUTO_INCREMENT,
+    `slug`                  varchar(255) NOT NULL UNIQUE,
```

Comment:
Note: It's possible that the slug may end up not being unique. This would be OK if we retrieve challenges by their ID, e.g. with the app route `/challenge/{id}/{slug}`.

Reply:
Are the new routes already implemented?  I will keep this as unique until then.

Reply:
We should have the new routes implemented later this month.

Reply:
cc @rrchai see my initial comment about above about the route design for challenges.

Reply:
Constraint has been removed.